src/NCronJob/Registry/JobDefinition.cs

```diff
@@ -15,6 +15,10 @@ internal sealed record JobDefinition(
 
     public CancellationToken CancellationToken { get; set; }
 
+    public bool IsOneTimeJob=> (CronExpression is null && RunAt != null) || IsStartupJob;
+    public DateTimeOffset? RunAt { get; set; }
```

Comment:
Is this still needed?

Reply:
Not particularly for this PR, but I added the same to another branch I'm working on for a bug fix related to instant jobs, so I just left it. I'll can remove it to not have to deal with merge issues.